management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -98,6 +102,38 @@
 
   CacheManager cacheManager;
 
+  @Test
+  public void descriptorClusteredStoreTest() {
+    ManagementRegistryService managementRegistry = new DefaultManagementRegistryService(new DefaultManagementRegistryConfiguration()
+            .setCacheManagerAlias("my-super-cache-manager"));
+
+    cacheManager = CacheManagerBuilder.newCacheManagerBuilder()
+        // cluster config
+        .with(ClusteringServiceConfigurationBuilder.cluster(URI.create("passthrough://server-1:9510/my-server-entity-1"))
+            .autoCreate()
+            .defaultServerResource("primary-server-resource"))
+        // management config
+        .using(managementRegistry)
+        // cache config
+        .withCache("cache-1", CacheConfigurationBuilder.newCacheConfigurationBuilder(
+            String.class, String.class,
+            newResourcePoolsBuilder()
+                .heap(10, EntryUnit.ENTRIES)
+                .with(ClusteredResourcePoolBuilder.clusteredDedicated("primary-server-resource", 1, MemoryUnit.MB)))
+            .build())
+        .build(true);
+
+    assertThat(managementRegistry.getCapabilities(), hasSize(5));
+    assertThat(new ArrayList<>(managementRegistry.getCapabilities()).get(0).getName(), equalTo("ActionsCapability"));
```

Comment:
`new ArrayList<>` is JDK 8+ only and this source code is supposed to be 1.6 compatible.